main.cpp

```diff
@@ -602,6 +701,18 @@ void FilterSection::displayInitState()
               << "Comb pattern index = " << combPatternIndex << std::endl;
 }
 
+void FilterSection::automateCut(float cutFreq)
+{
+    if (cutFreq < cutFrequency)
+    {
+        while (cutFrequency > cutFreq)
+            {
+                cutFrequency -= 100.0f;
+                std::cout << "Decreasing cut frequency to " << cutFrequency << std::endl;
+            }
```

Comment:
FIXME: indentation
curly braces should line up with the keywords on the line above
- [ ] revise